holidays/countries/germany.py

```
@@ -138,6 +153,31 @@ def _populate_public_holidays(self):
         if self.subdiv == "BYP":
             self._populate_subdiv_by_public_holidays()
 
+    def _populate_school_holidays(self):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_populate_school_holidays`

Add return type annotation: `None`

(ANN202)